server/src/test/java/org/elasticsearch/index/get/DocumentFieldTests.java

```diff
@@ -123,15 +123,22 @@ private static DocumentField mutateDocumentField(DocumentField documentField) {
                     DocumentField listField = new DocumentField(randomAlphaOfLength(5), listValues);
                     return Tuple.tuple(listField, listField);
                 case 2:
-                    List<Object> objectValues = randomList(1, 5, () ->
-                        Map.of(randomAlphaOfLength(5), randomInt(),
```

Comment:
The random keys I added here really aren't critical. We could instead just use fixed keys like "key1", "key2", "key3" to keep the test simple.